ossapi/ossapiv2.py

```diff
@@ -451,6 +452,26 @@ def beatmap(self, beatmap_id: int) -> Beatmap:
         """
         return self._get(Beatmap, f"/beatmaps/{beatmap_id}")
 
+    # /beatmapsets
+    # ------------
+
+    @request
+    def beatmapset_discussion_posts(self,
+        beatmapset_session_id: Optional[int] = None,
+        limit: Optional[int] = None,
+        page: Optional[int] = None,
+        sort: Optional[BeatmapDiscussionPostSort] = None,
```

Comment:
again this should be a new type `BeatmapDiscussionPostSortT` which is a union of BeatmapDiscussionPostSort and str. We do this for all forward-facing params which are enums so users can pass the raw string ("id_asc") instead of  BeatmapDiscussionPostSort.NEW. See https://github.com/circleguard/ossapi/pull/17/files#diff-ec81d0ffbef2ee0eac8fb17b8eee9048ef08ef11970a0f4899ecb9aadfd20f90R29-R42